src/content/rest/__tests__/restApi.test.ts

```diff
@@ -298,6 +313,39 @@ describe("rest api", () => {
       });
     });
 
+    describe("search contents", () => {
+      it("create content for search", async () => {
+        await create("products", {
+          title: "Search Me Product"
+        });
+
+        await create("news", {
+          slug: "News Search Slug",
+          title: "Search Me News"
+        });
+      });
+
+      it("should find content by search", async () => {
+        expect((await search("Search Me", false)).total).toBe(2);
+        expect((await search("News Search Slug", false)).total).toBe(1);
+        expect((await search("Search Me Product", false)).total).toBe(1);
+      });
+
+      it("should find limited content by search", async () => {
```

Comment:
This test could be more "documenting" if you'd use named variables as search parameters.

Reply:
Good point!